dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -129,7 +129,7 @@ public static void cleanStatePrivate2(DropwizardTestSupport<DockstoreWebserviceC
         LOG.info("Dropping and Recreating the database with confidential 2 test data");
         cleanStatePrivate2(support, CONFIG_PATH, isNewApplication);
         // TODO: it looks like gitlab's API has gone totally unresponsive, delete after recovery
```

Comment:
Change the comment to say something like, you can comment out the following line to disable gitlab
tool and workflow discovery